staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -46,6 +46,10 @@ import (
 	testingclock "k8s.io/utils/clock/testing"
 )
 
+const (
+	customEventFreshDuration = DefaultEventFreshDuration
```

Comment:
Don't know if custom prefix is needed. As this is test file, just keeping the name `eventFreshDuration` should be ok. Better would be pass it though newTestWatchCache and remove the const. 

Have you considered testing non `DefaultEventFreshDuration`, but a higher value?

Reply:
Change of the "event fresh duration" impacts the initially suggested sizes of the input and result channels within the watch cache - logic for their resizing is consistent no matter the initial value of that option, though.

I added some new test cases to `TestSuggestedWatchChannelSize` to take another possible value of that option into account.